elyra/pipeline/processor.py

```diff
@@ -69,3 +81,7 @@ def type(self):
     @abstractmethod
     def process(self, pipeline):
         raise NotImplementedError()
+
+    @abstractmethod
+    def export(self, pipeline):
+        raise NotImplementedError()
```

Comment:
Is export functionality required for all pipeline processors?  If not, this may need to be `pass`.  If we don't know yet, we can leave as NotImplementedError and just have a `pass` implementation on the new processor for the time being - or update this to `pass` at that time.

Reply:
I see this being required for both kfp and the currently in-progress airflow integration. Beyond that, im not sure of what other runtimes we are planning on looking at adding, we can change it to `pass` when we cross that bridge